src/scripts/modules/media/media.coffee

```diff
@@ -111,7 +114,7 @@ define (require) ->
       adjustHashTop = ->
         handleHeaderViewPinning()
         if isPinned
-          obscured = $pinnable.height()
+          obscured = $pinnable.height() + 50
```

Comment:
How well does this works when the window width is smaller (like on a mobile phone)?

If this number will continue to be used can it be moved into a common file so it is not duplicated?